tests/x11/multi_users_dm.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# This test checks if many users make the login manager hard to use
+# i.e. if it takes more than one click to access the username text field
+
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    my $users_to_create = 100;
+    my $encrypted_password = crypt($password, "abcsalt");
+
+    # login
+    select_console 'root-console';
+
+    # disable autologin
+    script_run "cp /etc/sysconfig/displaymanager /etc/sysconfig/displaymanager.back";
+    script_run "sed -i 's/^DISPLAYMANAGER_AUTOLOGIN.*\$/DISPLAYMANAGER_AUTOLOGIN=\"\"/' /etc/sysconfig/displaymanager";
+    script_run "/home/$username/data/create_users $users_to_create \"$encrypted_password\"";
+
+    type_string "systemctl isolate multi-user.target\n";
+    reset_consoles;
+    wait_idle 10;
```

Comment:
we want to get rid of wait_idle where possible as it is not portable. Can you replace this by an assert_screen or wait_still_screen or wait_serial?